aten/src/ATen/native/native_functions.yaml

```diff
@@ -14724,6 +14724,11 @@
   variants: function
   tags: nondeterministic_seeded
 
+- func: _scaled_dot_product_attention_math_for_mps(Tensor query, Tensor key, Tensor value, Tensor? attn_mask=None, float dropout_p=0.0, bool is_causal=False, Tensor? dropout_mask=None, *, float? scale=None) -> (Tensor, Tensor)
```

Comment:
Why add to native_functions.yaml

Reply:
So that I can compare against _scaled_dot_product_attention_math on Metal